.github/workflows/setup_pwa_manual.yml

```diff
@@ -0,0 +1,121 @@
+        default: "https://zzrf-002.dx.commercecloud.salesforce.com"
+      org_id:
+        type: string
+        description: Org Id
+        default: "f_ecom_zzrf_002"
+      short_code:
+        type: string
+        description: Short Code
+        default: "kv7kzm78"
+      client_id:
+        type: string
+        description: Client Id
+      site_id:
+        type: string
+        description: Site Id
+        default: "RefArch"
+      mrt_target_env:
+        type: string
+        description: MRT Targent Environment
+      is_private_client:
+        type: boolean
+        description: Is Private Client?
+        default: true
+      setup_hybrid:
+        type: boolean
+        description: Setup Phased Headless rollout?
+        default: false
+
+jobs:
+  setup-pwa-kit:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Initialize
+        id: initialize
+        run: |
+          set -e
+          echo "PROJECT_DIR=my-retail-react-app" >> $GITHUB_ENV
+
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        id: setup_node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          cache: "npm"
+
+      - name: Setup PWA dependencies
+        id: setup_pwa_dependencies
+        run: |-
+          set -e
+          sudo apt-get update -yq && sudo apt-get install --no-install-recommends python2 python3-pip time -yq
+          sudo pip install -U pip setuptools
+          sudo pip install awscli==1.18.85 datadog==0.40.1
+          node ./scripts/gtime.js monorepo_install npm ci
+          npm ci
+        shell: bash
+
+      - name: Generate app
+        id: generate_app
+        uses: "./.github/actions/generate_app"
+        with:
+          use_extensibility: ${{ github.event.inputs.use_extensibility }}
+          project_name: ${{ github.event.inputs.project_name }}
+          instance_url: ${{ github.event.inputs.instance_url }}
+          org_id: ${{ github.event.inputs.org_id }}
+          short_code: ${{ github.event.inputs.short_code }}
+          client_id: ${{ github.event.inputs.client_id }}
+          site_id: ${{ github.event.inputs.site_id }}
+          is_private_client: ${{ github.event.inputs.is_private_client }}
+          setup_hybrid: ${{ github.event.inputs.setup_hybrid }}
+          project_dir: ${{ env.PROJECT_DIR }}
+
+      - name: Create MRT credentials file
+        id: create_mrt_credentials
+        uses: "./.github/actions/create_mrt"
+        with:
+          mobify_user: ${{ secrets.MOBIFY_CLIENT_USER }}
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
+
+      - name: Create MRT target
+        id: create_mrt_target
+        uses: "./.github/actions/create_mrt_target"
+        with:
+          project_name: ${{ github.event.inputs.project_name }}
+          target_name: ${{ github.event.inputs.mrt_target_env }}
+          mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
```

Comment:
(Nit) I guess the idea with renaming `MOBIFY_CLIENT_API_KEY` => `mobify_api_key` is simply to avoid the confusion of the same name in many places? On the flip side, I find it a bit tough to find keys / variables when they don't all have the same name.

Reply:
Named this way to follow the convention -  global variables in UPPER_SNAKE_CASE and local variables in lower_snake_case